canonical/lxd-ui
# feat: update aside panels to use react portal

## Done

- Updated side panels in lxd to use react portal. This allows passing props down to the panels which will be needed for permissions management.

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Check instance details and profile details panels, they should work the same as before.
